dts/arm/microchip/sam/sam_d5x_e5x/common/samd5xe5x.dtsi

```diff
@@ -254,6 +254,38 @@
 				 <&gclkperiph CLOCK_MCHP_GCLKPERIPH_ID_SERCOM5_CORE>;
 			clock-names = "mclk", "gclk";
 		};
+
+		adc0: adc@43001c00 {
+			compatible = "microchip,adc-g1";
+			reg = <0x43001c00 0x4a>;
+			interrupts = <118 0>, <119 0>;
+			interrupt-names = "overrun", "resrdy";
+			/* 48 MHz GCLK / 4 prescaler = 12 MHz ADC clock. */
+			prescaler = <4>;
```

Comment:
prescaler is application dependent.
here we add only values that should never be changed from the user in the board.
drop `prescaler` and `vref-mv`.